src/shared-components/Card/ListItems/LisItemsTime.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+  Typography,
+} from '@mui/material';
+import { colors } from 'src/styles/colors';
+
+export const ListItemsTime = ({
+  title,
+  description,
+}: {
+  title: string;
+  description: string;
+}) => {
+  const currentTime = new Date().toLocaleTimeString([], {
+    hour: '2-digit',
+    minute: '2-digit',
+    hour12: false,
+  });
+  return (
+    <List>
+      <ListItem>
+        <ListItemText primary={title} secondary={description} />
+        <ListItemSecondaryAction>
+          <Typography color={colors.dark[25]}>{currentTime}</Typography>
```

Comment:
Do not hardcode time here, it should be passed as a prop when component is used.
To format the time in a specific way, it's better to add a function that formats any passed time in some generic place in the project (e.g. create utils folder inside src folder, then add timeUtils.js file to it).

Reply:
Fixed